code/modules/research/xenobiology/crossbreeding/stabilized.dm

```diff
@@ -66,35 +66,35 @@ Stabilized extracts:
 
 /obj/item/slimecross/stabilized/metal
 	colour = "metal"
-	effect_desc = "Adds a sheet of material to a stack in the owners backpack, it is based on a timer."
+	effect_desc = "Every 30 seconds, adds a sheet of material to a random stack in the owner's backpack."
 
 /obj/item/slimecross/stabilized/yellow
 	colour = "yellow"
-	effect_desc = "Every couple of seconds it recharges a device on the owner by 10%."
+	effect_desc = "Every ten seconds or few seconds it recharges a device on the owner by 10%."
```

Comment:
Pardon me, I meant to say choose between "ten seconds" and "few seconds". Just go with "ten seconds" imo.